### PR TITLE
license update

### PR DESCRIPTION
**Pull request by ordian**
extracted from #11542

### CODE REVIEW

File: rpc/src/v1/types/eth_types.rs [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+// (at your option) any later version.
+
+// Open Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Open Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
```
Comment (ordian) [inline rpc/src/v1/types/eth_types.rs]: how come this file missing a license header? :)

Comment (niklasad1) [reply rpc/src/v1/types/eth_types.rs]: It is probably my fault when I refactored to use `ethereum-types` in RPC and forgot apparentely.

**Comment by ordian** [comment]
> The license header in util/EIP-152/src/portable.rs is not updated, can you fix it?

Good catch! Somehow the script didn't fix it.

**Comment by niklasad1** [comment]
> Good catch! Somehow the script didn't fix it.

The script is hacky :)

I wrote it but assumes a new line after the license header which `util/EIP-152/src/portable.rs` don't have :eyes:

**Pull request merged**